pkg/openshift/webconsole.go

```diff
@@ -0,0 +1,63 @@
+package openshift
+
+//go:generate go run ./.packr/packr.go
+
+import (
+	"context"
+	"github.com/RHsyseng/operator-utils/pkg/logs"
+	"github.com/RHsyseng/operator-utils/pkg/utils/kubernetes"
+	"github.com/RHsyseng/operator-utils/pkg/utils/openshift"
+	"github.com/ghodss/yaml"
+	"github.com/gobuffalo/packr/v2"
+	hawtiov1alpha1 "github.com/hawtio/hawtio-operator/pkg/apis/hawtio/v1alpha1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var log = logs.GetLogger("openshift-webconsole")
+
+func ConsoleYAMLSampleExists() error {
+	gvk := schema.GroupVersionKind{Group: "console.openshift.io", Version: "v1", Kind: "ConsoleYAMLSample"}
+	return kubernetes.CustomResourceDefinitionExists(gvk)
+}
+
+func CreateConsoleYAMLSamples(c client.Client) {
+	log.Info("Loading CR YAML samples.")
+	box := packr.New("cryamlsamples", "../../deploy/crs")
+	if box.List() == nil {
+		log.Error("CR YAML folder is empty. It is not loaded.")
+		return
+	}
+	for _, filename := range box.List() {
+		yamlStr, err := box.FindString(filename)
+		if err != nil {
+			log.Info("yaml", " name: ", filename, " not created:  ", err.Error())
+			continue
+		}
+		hawtio := hawtiov1alpha1.Hawtio{}
+		err = yaml.Unmarshal([]byte(yamlStr), &hawtio)
+		if err != nil {
+			log.Info("yaml", " name: ", filename, " not created:  ", err.Error())
+			continue
+		}
+		productName := os.Getenv("PRODUCT_NAME")
```

Comment:
Do you have any objection replacing the environment variable by a build-time variable like we do here:

https://github.com/hawtio/hawtio-operator/blob/4183906c78c01df6e981de33682c3493cfcb51c1/pkg/controller/hawtio/hawtio_controller.go#L72-L74

I tend to think this is best address at build-time, rather than exposing that concern to the deployment as environment variable.

Reply:
@astefanutti 
Go build time variable added to openshift.ProductName. I used operator-sdk build cmd for development and testing.  
![hawtio](https://user-images.githubusercontent.com/42355256/82698064-1e610d80-9c38-11ea-8428-9aceb9ee7cbb.png)

The example you gave me was 'go build', I am sure how you build with "go build" cmd where you attached those build time variables and then create the final image for openshift deployment, so I didn't test this route. 
If I should test this route as well, please provide me the build and deployment steps.

Reply:
We do the Docker build manually in the product build using `go build`. I think it's safe assuming it will work as we already rely on build variables for other purposes.